schemas/url.yml

```diff
@@ -58,6 +58,9 @@
 
         In some cases a URL may refer to an IP and/or port directly, without a
         domain name. In this case, the IP address would go to the `domain` field.
+
+        If the URL contains a literal IPv6 address, `domain` may include the `[` and `]`
+        characters enclosing the address (https://www.ietf.org/rfc/rfc2732.txt[RFC 2732]).
```

Comment:
We can link when necessary, but the asciidoc links are also clunky in our descriptions, especially when they appear in artifacts other than the documentation.

I think we could get rid of this one and just say "IETF RFC 2732". Note that I'm adding an explicit mention of "IETF", because we now also have our own RFCs :-)

WDYT @ebeahan?

Reply:
++ sounds sensible.